.github/workflows/docker.yml

```diff
@@ -0,0 +1,72 @@
+name: "docker integration"
+
+on:
+  workflow_call:
+    inputs:
+      scenario:
+        type: string
+        description: molecule scenario
+        required: false
+        default: "false"
+      os_distribution:
+        type: string
+        description: OS image to use for the container
+        required: false
+        default: rockylinux:9
+      dtas_version:
+        type: string
+        description: git ref to use when cloning DTAS repo
+        required: false
+        default: master
+    secrets:
+      nexus_username:
+        required: false
+      nexus_password:
+        required: false
+      dtas_token:
+        required: false
+
+env:
+  DTAS_VERSION: ${{ inputs.dtas_version }}
+  BUILD_NUMBER: ${{ github.run_id }}
+  PY_COLORS: 1
+  PYTHONUNBUFFERED: 1
+
+jobs:
+  docker_integration:
+    name: Test ${{ inputs.scenario }} scenario on ${{ inputs.os_distribution }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # avoid auth conflict when cloning DTAS during verify step
+          persist-credentials: false
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        id: setup-python
+        with:
+          python-version-file: '.python-version'
+          cache: pipenv
+
+      - name: Install python packages
+        uses: Alfresco/alfresco-build-tools/.github/actions/pipenv@v3.4.0
+        with:
+          python-version: ${{ steps.setup-python.outputs.python-version }}
+
+      - name: Install and cache ansible galaxy dependencies
+        uses: ./.github/actions/galaxy
+        with:
+          cache-name: enterprise
```

Comment:
mmm switch to another cache name to avoid poisoning between community and enterprise jobs

Reply:
Why do you think it would poison the cache? the galaxy assets are identical for both ent. & com.

Reply:
I am talking about malicious stuff like the one described in https://scribesecurity.com/blog/github-cache-poisoning/

in general is better to keep separate caches for workflows which runs from forked repo and ours which has access to secrets - at the cost of increased cache usage which should not really have a big impact for us here